### DISCUSSION TITLE
config file for Weishaupt heating available?

### DISCUSSION THREAD

**Discussion by Lueghi** [john30/ebusd#582] [General]
I would like to connect my Weishaupt heating to my smarthome visualisation (ioBroker) with the ebusd. Is there a config file available for this heating?
Regards,
Stefan

**Comment by kalledausb** [comment]
I had good results with these config files for a Weishaupt WTC:
https://github.com/J0EK3R/ebusd-configuration-weishaupt

**Reply by Lueghi** [reply]
Thanks a lot. This looks fine. I will try / check it in detail.
